src/renderer/components/pages/Channel.tsx

```diff
@@ -16,7 +17,7 @@ const useStyles = makeStyles(() => ({
   }
 }))
 
-interface IChannelComponentProps {
+type IChannelComponentProps = ChannelHeaderProps & {
```

Comment:
let's revert back to `interface IChannelComponentProps` in this line, but let's change in line 30:

```
-export const Channel: React.FC<IChannelComponentProps> = ({ channelType, ...props }) => {
+export const Channel: React.FC<ChannelHeaderProps & IChannelComponentProps> = ({ channelType, ...props }) => {
```